waltz-tools/src/main/java/com/wepay/waltz/tools/cluster/ClusterCli.java

```diff
@@ -454,6 +564,19 @@ private StorageAdminClient openStorageAdminClient(String storageHost, Integer st
             return zkPartitionToReplicaMap;
         }
 
+        private Map<Integer, Set<String>> getZkPartitionToStorageNodeSetMapping(StoreMetadata storeMetadata) throws Exception {
```

Comment:
There seems to be two methods now for `getZkPartitionToStorageNode` Mapping: 
 - One with return type as `Map<Integer, Set<String>>`
 - Other with return type as `Map<Integer, List<String>>`

Only make use of one method and do the corresponding changes.

Reply:
Removed the method with `Map<Integer, List<String>>`, thanks for suggestion :)